jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -426,7 +430,11 @@ def _image_viewer_update(self, viewer, x, y):
                 arr = image.get_component(attribute).data
                 unit = image.get_component(attribute).units
                 if image.ndim == 3:
-                    value = arr[int(round(x)), int(round(y)), viewer.state.slices[-1]]
+                    if self.app.config == 'lcviz':
+                        value = arr[viewer.state.slices[0], int(round(y)), int(round(x))]
+                    else:
+                        # cubeviz case:
+                        value = arr[int(round(x)), int(round(y)), viewer.state.slices[-1]]
```

Comment:
if we store the indices in the helper, is there any way to generalize this line? 🤔

Reply:
Helpers could have hidden `__getitem__`-like methods for indexing the source data, right?

Reply:
ummm, we could maybe monkeypatch it on to the data-layer, but I'm not sure helper-level makes sense since helpers can have multiple types of data each with their own dimensionality, etc.